<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnsPage</class>
 <widget class="QWidget" name="ColumnsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gb_showColumns">
     <property name="title">
      <string>Hidden columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_showFilename">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showExtension">
        <property name="text">
         <string>Extension</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showSize">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showTime">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showAttr">
        <property name="text">
         <string>Attributes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
